static/app/views/replays/detail/layout.tsx

```diff
@@ -0,0 +1,272 @@
+import BaseResizePanel from 'react-resize-panel';
+import styled from '@emotion/styled';
+
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import ReplayTimeline from 'sentry/components/replays/breadcrumbs/replayTimeline';
+import ReplayView from 'sentry/components/replays/replayView';
+import space from 'sentry/styles/space';
+import useFullscreen from 'sentry/utils/replays/hooks/useFullscreen';
+import theme from 'sentry/utils/theme';
+import Breadcrumbs from 'sentry/views/replays/detail/breadcrumbs';
+import FocusArea from 'sentry/views/replays/detail/focusArea';
+import FocusTabs from 'sentry/views/replays/detail/focusTabs';
+
+type Layout =
+  /**
+   * ### Sidebar
+   * ┌───────────────────┐
+   * │ Timeline          │
+   * ├──────────┬────────┤
+   * │ Details  > Video  │
+   * │          >        │
+   * │          >^^^^^^^^┤
+   * │          > Crumbs │
+   * │          >        │
+   * └──────────┴────────┘
+   */
+  | 'sidebar'
+  /**
+   * ### Topbar
+   *┌────────────────────┐
+   *│ Timeline           │
+   *├───────────┬────────┤
+   *│ Video     │ Crumbs │
+   *│           │        │
+   *├^^^^^^^^^^^^^^^^^^^^┤
+   *│ Details            │
+   *│                    │
+   *└────────────────────┘
+   */
+  | 'topbar';
+
+const GrabberColor = encodeURIComponent(theme.gray300);
+const GrabberSVG =
+  `url('data:image/svg+xml,` +
+  `%3Csvg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" fill="${GrabberColor}" height="16px" width="16px"%3E` +
+  '%3Ccircle cx="4.73" cy="8" r="1.31"%3E%3C/circle%3E' +
+  '%3Ccircle cx="4.73" cy="1.31" r="1.31"%3E%3C/circle%3E' +
+  '%3Ccircle cx="11.27" cy="8" r="1.31"%3E%3C/circle%3E' +
+  '%3Ccircle cx="11.27" cy="1.31" r="1.31"%3E%3C/circle%3E' +
+  '%3Ccircle cx="4.73" cy="14.69" r="1.31"%3E%3C/circle%3E' +
+  '%3Ccircle cx="11.27" cy="14.69" r="1.31"%3E%3C/circle%3E' +
+  '%3C/svg%3E' +
+  `')`;
```

Comment:
This is the generated SVG from https://github.com/getsentry/sentry/blob/master/static/app/icons/iconGrabbable.tsx

I couldn't sort out how to extract it from the react component. I think it could be done react-dom-server or to render it inside an unmounted dom node then copy the html content. All that seemed slower to build and slower to exec compared to having an encoded svg.